app/scripts/museum/museum-service.js

```diff
@@ -100,6 +106,30 @@
             return Util.makeRequest(sql, query);
         }
 
+        function listByCity(params) {
+            var attributes = { 'gcity': 'city', 'gstate': 'state', 'gzip': 'zip' };
```

Comment:
There are two different search cases via the geocoder that we want to handle:
1. User searches a city and/or state --> Filter results to matching city and state
2. User searches a zip code --> Filter results to matching zip code. In this case I suppose the additional search on city/state is somewhat useless, and this could be updated to just pass zip if its provided, and not also search on city/state.